prepare_zephir_records/cid_minter/zephir_cluster_lookup.py

```diff
@@ -31,20 +30,78 @@ def construct_select_zephir_cluster_by_cid(cids):
 
     return SELECT_ZEPHIR_BY_OCLC + " " + AND_CID_IN + " (" + cids + ") " + ORDER_BY
 
+def construct_select_zephir_cluster_by_contribsys_id(contribsys_ids):
```

Comment:
Good question. The invalid_sql_in_clause_str was created to make sure the oclc numbers came in in the 'ocn1', 'ocn2' format. The OCNs were passed as integer initially. I will if I can make the invalid_sql_in_clause_str function more generic. Thank you!

Reply:
Okay I see now! It might be nice to just check that its a comma separated string, not critical though. At first glance it just looked like the `contribsys_ids` argument might be a list, but a list couldn't be concatenated with the SQL string. This all looks good though, I'm approving!